@here/harp-webpack-utils/scripts/HarpWebpackConfig.js

```diff
@@ -96,11 +110,14 @@ export function addHarpWebpackConfig(config?: Configuration, harpConfig?: HarpWe
     return bundles;
 }
 
-function createPlugins(htmlTemplate?: string): Plugin[] {
+function createPlugins(htmlTemplate /*?: string*/) /*: Plugin[]*/ {
     const plugins = [
         new CopyWebpackPlugin([
             {
-                from: "node_modules/@here/harp-map-theme/resources",
+                from: path.join(
+                    path.dirname(require.resolve("@here/harp-map-theme/package.json")),
+                    "resources"
+                ),
                 to: "resources",
```

Comment:
General.
Although, we commonly use `resources` in our demo/example apps, here we're in namespace of user application and i think we should create all names in such way, so we avoid potential clashes, so this should be something like
`harp-resources` or just `harp-map-theme`.

@robertoraggi It would be good, if this "default" naming scheme would be discussed with above concern, so defaults for `font://`, `poi://` are aligned with tools like this one.